examples/hosted-delivery-function-potterdb-rest-api/delivery-functions/index.ts

```diff
@@ -0,0 +1,94 @@
+import type { DeliveryFunctionEventHandler as EventHandler } from '@contentful/node-apps-toolkit'
+import { createSchema, createYoga } from 'graphql-yoga'
+import { typeDefs } from './schema'
+import { NodeRequest } from '@whatwg-node/server/typings/utils'
+import { GraphQLError } from 'graphql'
+
+const schema = createSchema({
```

Comment:
Personally, to more easily understand what is going on for GraphQL novices, I would put the schema definition and resolvers in the same file (e.g. move the `createSchema` call to `schema.ts`